.github/shared/src/swagger.js

```diff
@@ -42,6 +56,18 @@ export class Swagger {
   /** @type {Tag | undefined} Tag that contains this Swagger */
   #tag;
 
+  /** @type {Operation[]} map of the operations in this swagger*/
+  #operations;
+
+  /** @type {string}  swagger version*/
+  #version;
+
+  /** @type {string} version kind*/
+  #versionKind;
+
+  /** * @type {string} file name of the swagger file, e.g. "LongRunningBackend.json" */
+  #fileName;
```

Comment:
Does this need to be a separate property, or can it be extracted from the existing #path property?

Reply:
I removed them and added get functions to compute them.